src/Microsoft.VisualStudio.Editors/ResourceEditor/ResourceStringTable.vb

```diff
@@ -1869,6 +1870,14 @@ Namespace Microsoft.VisualStudio.Editors.ResourceEditor
             End Sub
         End Class
 
+        Protected Overrides Function ProcessDataGridViewKey(e As KeyEventArgs) As Boolean
+            If e.KeyCode = Keys.Space AndAlso e.Shift Then
+                Return False
+            Else
+                Return MyBase.ProcessDataGridViewKey(e)
```

Comment:
What does the DataGridView normally treat Shift+Space as?  Will this change break expectations for something people expect from DGV in other applications?

Reply:
As far as I can tell, Shift+Space selects either the entire row, or the entire column, depending on the value of `DataGridView.SelectionMode`. In practice, this highlights the entire row, without losing the edit focus. This has no practical benefit (to me, at least), and it also eats the space input, which is extremely annoying when typing text with the spacebar held down (to type all-caps text, or to type an ampersand between words, for example). I have never heard of anyone actually using this "feature" anywhere, and in fact I'd reckon that other DataGridView users have become just as confused by the same quirk that the resx editor suffers from.

Reply:
I think I'd prefer this to be limited to the only when editing the content of the cells, there's no reason to eat CTRL+Space when not in editing mode. This lets you easily select and delete a cell when you moving the selection around for a single cell.

Reply:
It looks like DataGridView.IsCurrentCellInEditMode indicates that.